blueprints/factories/bigquery-factory/variables.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "views_dir" {
+  description = "Relative path for the folder storing view data."
+  type        = string
+}
+
+variable "tables_dir" {
+  description = "Relative path for the folder storing table data."
+  type        = string
+}
+
+variable "project_id" {
```

Comment:
one extra comment, and please feel free to resolve this comment and implement this after a first merge. It would be nice using the project module with `project_create = false` to configure the project, activating needed APIs and potentially granting IAM roles to relevant groups.

Reply:
Thanks for the suggestion, I will have a look at that after the first merge!